entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/impl/BaseEntity.java

```diff
@@ -165,7 +165,6 @@ public void setIdentifier(String[] identifier) {
 		this.identifier = identifier;
 	}
 
-	@JsonIgnore
```

Comment:
I think that JsonIgnore is still needed for the json serialization. Can we move the @JacksonXmlProperty annotation to the getEntityId() method?